Config.cmake/runtime_test.cmake

```diff
@@ -11,79 +11,54 @@
 #    You should have received a copy of the FMILIB_License.txt file
 #    along with this program. If not, contact Modelon AB <http://www.modelon.com>.
 
-set(RTTESTDIR ${FMILIBRARYHOME}/Test)
+set(FMIL_TEST_DIR ${FMILIBRARYHOME}/Test)
+set(CATCH_INCLUDE_DIR ${FMILIB_THIRDPARTYLIBS}/Catch2/single_include/catch2)
+set(TEST_OUTPUT_FOLDER ${CMAKE_CURRENT_BINARY_DIR}/Testing)
+set(CATCH_INCLUDE_DIR "${FMILIB_THIRDPARTYLIBS}/Catch2/single_include/catch2/")
```

Comment:
Mainly moved to top of file for readability, and renamed RTTESTDIR -> FMIL_TEST_DIR.